helm/calico-policy-only/templates/calico-policy-only.yaml

```diff
@@ -60,28 +60,6 @@ data:
       ]
     }
 ---
-
-
-# This manifest creates a Service, which will be backed by Calico's Typha daemon.
-# Typha sits in between Felix and the API server, reducing Calico's load on the API server.
-
-apiVersion: v1
-kind: Service
-metadata:
-  name: calico-typha
-  namespace: kube-system
-  labels:
-    k8s-app: calico-typha
-spec:
-  ports:
-    - port: 5473
-      protocol: TCP
-      targetPort: calico-typha
-      name: calico-typha
-  selector:
-    k8s-app: calico-typha
----
-
```

Comment:
i cant find reason why we should have the typha service here since we don't use it and don't have it anywhere